uae_reset() now wants some argument!

Antonio Ospite <ospite@studenti.unina.it>

diff -pruN uae-0.8.25___orig/src/ncurses.c uae-0.8.25/src/ncurses.c
--- uae-0.8.25___orig/src/ncurses.c	2006-06-07 17:41:49.000000000 +0200
+++ uae-0.8.25/src/ncurses.c	2006-06-07 17:42:33.000000000 +0200
@@ -596,7 +596,7 @@ void handle_events(void)
 	if (ch == 16) --lastmy; /* ^P */
 	if (ch == 11) {buttonstate[0] = keydelay;ch = 0;} /* ^K */
 	if (ch == 25) {buttonstate[2] = keydelay;ch = 0;} /* ^Y */
-	if (ch == 15) uae_reset (); /* ^O */
+	if (ch == 15) uae_reset (0); /* ^O */
 	if (ch == 17) uae_quit (); /* ^Q */
 	if (ch == KEY_F(1)) {
 	  curses_insert_disk();
diff -pruN uae-0.8.25___orig/src/NeXTwin.m uae-0.8.25/src/NeXTwin.m
--- uae-0.8.25___orig/src/NeXTwin.m	2006-06-07 17:41:49.000000000 +0200
+++ uae-0.8.25/src/NeXTwin.m	2006-06-07 17:42:15.000000000 +0200
@@ -86,7 +86,7 @@ static int keycode2amiga(NXEvent * theEv
 @implementation AmigaView
 -reset:sender
 	{
-	uae_reset();
+	uae_reset(0);
 	//m68k_reset();
 	return self;
 	}
diff -pruN uae-0.8.25___orig/src/od-beos/beos.cpp uae-0.8.25/src/od-beos/beos.cpp
--- uae-0.8.25___orig/src/od-beos/beos.cpp	2006-06-07 17:41:49.000000000 +0200
+++ uae-0.8.25/src/od-beos/beos.cpp	2006-06-07 17:43:48.000000000 +0200
@@ -719,7 +719,7 @@ void handle_events(void)
 	uint32 mouse_buttons;
 
 	if (reset_thyself) {
-		uae_reset();
+		uae_reset(0);
 		reset_thyself = false;
 	}
 
diff -pruN uae-0.8.25___orig/src/od-win32/keyboard.c uae-0.8.25/src/od-win32/keyboard.c
--- uae-0.8.25___orig/src/od-win32/keyboard.c	2006-06-07 17:41:49.000000000 +0200
+++ uae-0.8.25/src/od-win32/keyboard.c	2006-06-07 17:44:04.000000000 +0200
@@ -489,8 +489,8 @@ int my_kbd_handler (int keycode, int sca
 	record_key ((akey << 1) + 1);
 
     /* "Affengriff" */
-	if(  (keystate[AK_CTRL] || keystate[AK_RCTRL] ) && keystate[AK_LAMI] && keystate[AK_RAMI])uae_reset();
-    //if(  (keystate[AK_CTRL] || keystate[AK_RCTRL] ) && keystate[AK_RAMI])uae_reset();
+	if(  (keystate[AK_CTRL] || keystate[AK_RCTRL] ) && keystate[AK_LAMI] && keystate[AK_RAMI])uae_reset(0);
+    //if(  (keystate[AK_CTRL] || keystate[AK_RCTRL] ) && keystate[AK_RAMI])uae_reset(0);
 
     return 0;
 }
diff -pruN uae-0.8.25___orig/src/od-win32/win32gui.c uae-0.8.25/src/od-win32/win32gui.c
--- uae-0.8.25___orig/src/od-win32/win32gui.c	2006-06-07 17:41:49.000000000 +0200
+++ uae-0.8.25/src/od-win32/win32gui.c	2006-06-07 17:44:16.000000000 +0200
@@ -2261,7 +2261,7 @@ static BOOL CALLBACK MiscDlgProc (HWND h
 	    }
 	    break;
 	case IDC_RESETAMIGA:
-	    uae_reset();
+	    uae_reset(0);
 	    break;
 	case IDC_QUITEMU:
 	    uae_quit();
diff -pruN uae-0.8.25___orig/src/sdlgfx.c uae-0.8.25/src/sdlgfx.c
--- uae-0.8.25___orig/src/sdlgfx.c	2006-06-07 17:41:49.000000000 +0200
+++ uae-0.8.25/src/sdlgfx.c	2006-06-07 17:42:57.000000000 +0200
@@ -875,7 +875,7 @@ void handle_events (void)
 
     /* Handle UAE reset */
     if ((keystate[AK_CTRL] || keystate[AK_RCTRL]) && keystate[AK_LAMI] && keystate[AK_RAMI])
-	uae_reset ();
+	uae_reset (0);
 }
 
 int check_prefs_changed_gfx (void)
diff -pruN uae-0.8.25___orig/src/svga.c uae-0.8.25/src/svga.c
--- uae-0.8.25___orig/src/svga.c	2006-06-07 17:41:49.000000000 +0200
+++ uae-0.8.25/src/svga.c	2006-06-07 17:43:17.000000000 +0200
@@ -439,7 +439,7 @@ static void my_kbd_handler (int scancode
 
     /* "Affengriff" */
     if ((keystate[AK_CTRL] || keystate[AK_RCTRL]) && keystate[AK_LAMI] && keystate[AK_RAMI])
-	uae_reset ();
+	uae_reset (0);
 }
 
 static void leave_graphics_mode (void)
diff -pruN uae-0.8.25___orig/src/tui.c uae-0.8.25/src/tui.c
--- uae-0.8.25___orig/src/tui.c	2006-06-07 17:41:49.000000000 +0200
+++ uae-0.8.25/src/tui.c	2006-06-07 17:43:32.000000000 +0200
@@ -684,7 +684,7 @@ static int do_gui (int mode)
 	     case 0: DiskOptions (); break;
 	     case 1: OtherOptions (); break;
 	     case 2: save_settings (); break;
-	     case 3: uae_reset (); break;
+	     case 3: uae_reset (0); break;
 	     case 4: uae_quit (); break;
 	    }
 	}
